### ISSUE TITLE
Replace 240V element in the hot water storage tank

### ISSUE THREAD

**Issue by smr547** [CanberraOceanRacingClub/namadgi3#505]
This problem occurs only while on shore power. The circuit breaker in the port cabin lazarette locker (behind aft bulkhead) needs resetting when it trips. Peter Smith at Batemen's Bay is sourcing a new element for the HW system and will advise when available.

@pjwain and @SimonRD will be interested

**Comment by smr547** [comment]
Sigmar Marine hot water tank

https://www.quickitaly.com/sigmar/

![hot_watr_tank](https://user-images.githubusercontent.com/4327895/160969419-d2db351b-c29f-4e85-a6a7-
9a6f7a14d596.jpg)

**Comment by smr547** [comment]
We can source a new element for $245 and can replace it ourselves. Order placed today.

https://www.aquavolt.com.au/category/5341-water-heaters

**Comment by smr547** [comment]
New element has arrive -- replace at next working bee or in Coomera

**Comment by smr547** [comment]
The new heating element has not yet been installed. It is stored in a plastic bag (or a box) next to the water heater -- under the bunk in the port aft cabin.

**Comment by smr547** [comment]
Replaced by [Damien Hadzic](https://contacts.google.com/person/c5321404824969117156) -- job well done. Closing this issue.

**Comment by smr547** [comment]
It seems the heating element has failed again. Re-opening this issue
#budget=$500

**Comment by smr547** [comment]
New element ordered

**Comment by smr547** [comment]
Before attempting to install the new element, I tested the resistance across the plug connection at the power circuit — found it open circuit. Removed the thermostat and tested the heater element terminal. - about 60 ohms (normal for a working element). Inspection of the thermostat revealed a heavily corroded terminal post on the active (L) side where it mates with the element. Also suspicious burn marks near L terminal suggest an earth leakage at some time in the past. Cleaned all terminals thoroughly and reassembled. Tested power plug and observed nominal element resistance. Will test when back on shore power. 

![139F9021-029E-4773-A022-E6EC5EAF1DCD](https://github.com/CanberraOceanRacingClub/namadgi3/assets/4327895/09bc1e72-f4fb-4049-bc12-c25771aec8d2)
![D26954C3-8104-4D23-9EEC-5BA78B2EDF75](https://github.com/CanberraOceanRacingClub/namadgi3/assets/4327895/be49918f-b876-42fb-a4dd-772aedac66b8)
![0A0D199B-9220-41EF-AF03-F44775278C71](https://github.com/CanberraOceanRacingClub/namadgi3/assets/4327895/0cee0fb2-2ad3-4588-9603-052d0dd1e729)
